src/utils/response.js

```diff
@@ -0,0 +1,18 @@
+// Response function for error / success returns
```

Comment:
A response object was previously added to the project.

Reply:
I'll remove the redundant response object